mapit_gb/management/commands/mapit_UK_import_boundary_line.py

```diff
@@ -59,14 +59,11 @@ def handle_label(self, filename, **options):
             ons_code = feat['CODE'].value if feat['CODE'].value not in ('999999', '999999999') else None
             unit_id = str(feat['UNIT_ID'].value)
             area_code = feat['AREA_CODE'].value
-            if hasattr(control, 'patch_boundary_line'):
-                patch = control.patch_boundary_line(name, ons_code, area_code)
```

Comment:
I'm probably being dense here, but going through the history, did any control file modules ever have a `patch_boundary_line` function? (It clearly looks safe to remove, of course, I just wondered if it was ever used.)